Doc/license.rst

```diff
@@ -1040,3 +1040,30 @@ https://www.w3.org/TR/xml-c14n2-testcases/ and is distributed under the
    THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+asyncio
+----------
+
+The :mod:`asyncio` module contains the following notice::
```

Comment:
```suggestion
Parts of the :mod:`asyncio` module are incorporated from uvloop 0.16,
which is distributed under the MIT license::
```

Reply:
Good idea! Changed in https://github.com/aloisklink/cpython/commit/d5aee035ac4e0b7e5e6340e2c53d893d4eb90ba4 (I've also made `uvloop 0.16` a hyperlink)